ch_pipeline/analysis/flagging.py

```diff
@@ -117,7 +117,10 @@ def process(self, data) -> Union[containers.RFIMask, dcontainers.RFIMask]:
         # Create container to hold output
         if self.use_draco_container:
             # draco RFIMask container is not distributed
```

Comment:
I had a look into how to type this, and I can't really come up with a way to do it (since it isn't conditional on input type). It's maybe a suggestion that we should refactor somehow, but I'm not sure if that's really worth it for a fairly basic task

Reply:
Yeah. I think the best you can really do is have the output be the Union of all possible types.

For me the most useful thing is editor autocompletion, and I think all of those types have a pretty similar interface so it shouldn't really matter too much.